HeterogeneousCore/AlpakaMath/interface/deltaPhi.h

```diff
@@ -0,0 +1,36 @@
+#ifndef HeterogeneousCore_AlpakaMath_deltaPhi_h
+#define HeterogeneousCore_AlpakaMath_deltaPhi_h
+
+#include <alpaka/alpaka.hpp>
+
+namespace cms::alpakatools {
+
+  // reduce to [-pi,pi]
+  template <typename TAcc, typename T>
+    requires std::is_floating_point_v<T>
+  ALPAKA_FN_HOST_ACC ALPAKA_FN_INLINE T reduceRange(TAcc const& acc, T x) {
```

Comment:
`reduceRange` is not a good name, because it doesn't convey any relation to the [-π, π].

Reply:
maybe `reducePhiRange` would be better ?

Reply:
I agree, and this was brought up by Matti as well. However, in this PR, I tried to stick to the current CMSSW convention which uses `reduceRange` in multiple places for keeping φ within [-π, π]:
https://github.com/search?q=repo%3Acms-sw%2Fcmssw%20reduceRange&type=code
Do we want to change just for this version of this function?

Maybe a global renaming to something clearer (`reducePhiRange` sounds good to me) would make sense. If at least some of us think it's useful and it doesn't turn out to be a rabbithole, I could try to propose it in a different PR for wider review. But I wouldn't connect it to this PR.